src/shogun/machine/RandomForest.h

```diff
@@ -137,6 +137,12 @@ class RandomForest : public BaggingMachine
 	 * @return number of randomly chosen features during each node split
 	 */
 	int32_t get_num_random_features() const;
+	/** get feature importances of previous trained, use Mean Decrease
+	 * Impurity(MDI)
+	 *
+	 * @return arrays of feature importance
+	 */
+	SGVector<float64_t> get_feature_importances();
```

Comment:
If you don't cache the feature importances you could make this `const`, no?

Reply:
ok,i will change it :)